Adding test for issue 99

ipeirotis opened the pull request:
Adding test for #99

silvavn commented:
I had implemented this fix on #98 
I am not sure the testing is necessary.
If you simple search for the `gsh_small` as in f51ebca we will have html garbage in the abstract.

ipeirotis commented:
> I had implemented this fix on #98
> I am not sure the testing is necessary.
> If you simple search for the `gsh_small` as in [f51ebca](https://github.com/OrganicIrradiation/scholarly/commit/f51ebca888f0a92f9ebfff340eb6464068b2221e) we will have html garbage in the abstract.

Dropped the code for the fix, as your fix was better. 

I just added the test now, to ensure that the abstract field gets filled properly.
